src/navigation/stacks/auth/index.tsx

```diff
@@ -0,0 +1,17 @@
+import { StackParamList } from 'navigation/types';
+import { AppScreens } from 'navigation/types';
+import React from 'react';
```

Comment:
if I remember correctly I had an action where the imports where automatically fixed to avoid any inconsistency